<?xml version="1.0" encoding="UTF-8"?><testrun name="JDBCTests1" project="ex00" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="org.taesan.persistence.JDBCTests1" time="3.017">
    <testcase name="testConnectiont" classname="org.taesan.persistence.JDBCTests1" time="3.017">
      <error>java.sql.SQLException: Listener refused the connection with the following error:
ORA-12519, TNS:no appropriate service handler found
 &#13;
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)&#13;
	at oracle.jdbc.driver.PhysicalConnection.&lt;init&gt;(PhysicalConnection.java:553)&#13;
	at oracle.jdbc.driver.T4CConnection.&lt;init&gt;(T4CConnection.java:254)&#13;
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)&#13;
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)&#13;
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)&#13;
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)&#13;
	at org.taesan.persistence.JDBCTests1.testConnectiont(JDBCTests1.java:23)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:768)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
Caused by: oracle.net.ns.NetException: Listener refused the connection with the following error:
ORA-12519, TNS:no appropriate service handler found
 &#13;
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:399)&#13;
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)&#13;
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)&#13;
	... 30 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
